Graphs/Table/index.js

```diff
@@ -235,6 +287,25 @@ class Table extends AbstractGraph {
         this.updateData();
     }
 
+    // update column dynamically according to the value passed through the context
+    getColumnByContext(columns) {
+        const { context } = this.props;
+        const filterColumns = []
+        columns.forEach((d, index) => {
```

Comment:
please re-use the static method above!

Reply:
logic here seems to be different from that in static method. this function seems to return columns that satisfy the displayOption filter. ill recheck.

Reply:
removed this function